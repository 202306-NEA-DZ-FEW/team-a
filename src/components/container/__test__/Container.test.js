import renderer from "react-test-renderer";

import Container from "..";

it("renders correctly", () => {
  const tree = renderer.create(<Container />).toJSON();
  expect(tree).toMatchSnapshot();
});
