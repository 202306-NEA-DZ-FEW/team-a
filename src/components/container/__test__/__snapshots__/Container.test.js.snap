// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="max-w-6xl mx-auto px-8 xl:px-0 undefined"
/>
`;
